libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -266,8 +266,9 @@ class YulUtilFunctions
 	std::string storageArrayPushZeroFunction(ArrayType const& _type);
 
 	/// @returns the name of a function that will clear the storage area given
-	/// by the start and end (exclusive) parameters (slots).
-	/// signature: (start, end)
+	/// by the start position and number of slots to clear. The start position is in terms of storage slots and we
+	/// assume that the beginning of the clear range starts at the beginning of the start slot.
+	/// signature: (start, slotCount)
 	std::string clearStorageRangeFunction(Type const& _type);
```

Comment:
This should also mention that the function assumes that `slotCount` is a multiple of `_type.storageSize()`. When it's not, the function may clear more than it should, because it does it in multiples of `_type.storageSize()`.